aep/general/0131/aep.md.j2

```diff
@@ -10,9 +10,11 @@ resource.
 
 ## Guidance
 
- - APIs **must** provide a get method for resources. The purpose of the get method
-is to return data from a single resource.
- - Some resources take longer to be retrieved than is reasonable for a regular API request. In this situation, the API should use a [long-running operation](/long-running-operations).
+- APIs **must** provide a get method for resources. The purpose of the get
```

Comment:
nit: since this is just formatting (and not required as prettier passed before), it would be great to pick these up in a separate PR in the future.